packages/angular/cli/commands/e2e-impl.ts

```diff
@@ -14,8 +14,19 @@ import { Schema as E2eCommandSchema } from './e2e';
 export class E2eCommand extends ArchitectCommand<E2eCommandSchema> {
   public readonly target = 'e2e';
   public readonly multiTarget = true;
+  public readonly missingTargetError = `
+Cannot find "e2e" target for the specified project.
 
-  public async run(options: E2eCommandSchema & Arguments) {
-    return this.runArchitectTarget(options);
+You should add a package that implements e2e capabilities.
+
+Here are some options that are available:
+  1. TestCafe (https://testcafe.io/documentation/402793/recipes/integrations/use-angular-cli-builder)
+  2. WebdriverIO (https://webdriver.io/docs/protractor-migration/)
```

Comment:
Should we mentioned this? Since it cannot be used with this command?

Reply:
maybe like here: https://github.com/webdriverio/webdriverio-schematics

Reply:
Removed TestCafe for now, opened https://github.com/politie/angular-testcafe/issues/63 to ask them to add schematic.
Mentioned `ng add` for WebdriverIO.